webview/src/experiments/components/table/TableHead.tsx

```diff
@@ -86,21 +86,22 @@ export const TableHead = ({
       type: MessageFromWebviewType.REORDER_COLUMNS
     })
   }
-  const [ref, needsShadow] = useInView({
-    root,
-    rootMargin: '-15px 0px 0px 0px',
-    threshold: 1
-  })
 
   const selectedForPlotsCount = getSelectedForPlotsCount(rows)
 
   const firstExpColumnCellId = headerGroups[0].headers[0].id
 
+  const wrapper = useRef<HTMLDivElement>(null)
```

Comment:
Hooks are usually used at the start of a component. It may not happen here, but it's a good practice to avoid things like "rendered more hooks than during the previous render". This also includes the `useEffect` below. (I think that comes from the branch I pushed that we did while pairing, we didn't go back and clean stuff, oops sorry)

Reply:
Whoops! Thanks for spotting that!